docs/en/usage/apollo-user-guide.md

```diff
@@ -1 +1,501 @@
+
+> If you use Spring Security simple authentication provided by Apollo, you must remember to change the password of super administrator apollo
+
+### 7.1.2 Authorization
+
+Apollo supports fine-grained permissions control, so please make sure to control the permissions according to the actual situation: 1.
+
+1. [Project administrator privileges](#_121-Project-administrator-privileges)
+   * Apollo allows all logged-in users to create projects by default. If you only allow some users to create projects, you can turn on [Create project permission control](en/deployment/distributed-deployment-guide#_3110-role.create-application.enabled---whether-to-enable-control-over-creating-project-permissions)
+2. [Configure editing and publishing privileges](#_122-Configuring-editing-and-publishing-permissions)
+   * Configuration editing and publishing privileges support configuration by environment, for example, the development environment developers can complete the process of configuration editing and publishing by themselves, but the production environment publishing privileges to the test or operation and maintenance personnel
+   * It is recommended to turn on [release audit](en/deployment/distributed-deployment-guide#_322-namespace.lock.switch---Only-one-person-can-modify-the-switch-at-a-time-for-release-review) at the same time for production environment, so as to control that only one person can modify a configuration release and another person can release it. This ensures that configuration changes are adequately checked.
+3. [Configuration view permissions](#_61-Configure-view-permissions)
+   * You can specify that only project members of an environment are allowed to view the configuration of a private Namespace, thus avoiding sensitive configuration leaks, such as production environments
+
+### 7.1.3 System access
+
+In addition to user permissions, system access also needs to be considered in terms of.
+
+1. `apollo-configservice` and `apollo-adminservice` are designed based on the intranet trusted network, so for security reasons, `apollo-configservice` and `apollo-adminservice` are prohibited from being exposed directly to the public network
+2. For sensitive configurations, consider enabling [access secret key](#_62-Configuring-access-keys) so that only authenticated clients can access sensitive configurations
+3. 1.7.1 and above can consider enabling [access control](en/deployment/distributed-deployment-guide?#_326-admin-services.access.control.enabled---Configure-whether-apollo-adminservice-has-access-control-enabled) for `apollo-adminservice`, so that only [controlled](en/deployment/distributed-deployment-guide#_3112-admin-services.access.token---set-the-access-token-required-by-apollo-portal-to-access-the-apollo-adminservice-for-each-environment) `apollo-portal` can access the corresponding interface to enhance security
```

Comment:
```suggestion
3. 1.7.1 and above can consider enabling [access control](en/deployment/distributed-deployment-guide?#_326-admin-servicesaccesscontrolenabled-configure-whether-apollo-adminservice-has-access-control-enabled) for `apollo-adminservice`, so that only [controlled](en/deployment/distributed-deployment-guide#_3112-admin-servicesaccesstokens-set-the-access-token-required-by-apollo-portal-to-access-the-apollo-adminservice-for-each-environment) `apollo-portal` can access the corresponding interface to enhance security
```